### DISCUSSION TITLE
Hopefully I'm not overlooking something obvious?

### DISCUSSION THREAD

**Discussion by hexwitch** [mocaccinoOS/mocaccino#78] [General] [answered]
I am trying to use Mocaccino as a live usb.  I have etched the minimal desktop onto a usb but get as far as it asking me for a login.  I don't have it in front of me now, but I think it says that login is 'root' (automatic login).  Suffice to say that this doesn't work and I can't find a default login anywhere.

Am I missing something obvious here?  I'm assuming that it does run as live system?

Any help appreciated.

**Comment by mudler** [comment, marked as answer]
Hi, can you specify which version did you downloaded? the minimal flavor does not ship any DE by default.
